app/models/member.rb

```diff
@@ -25,6 +25,7 @@ class Member < ActiveRecord::Base
   has_one :braintree_customer,     class_name: 'Payment::Braintree::Customer'
   has_one :authentication, class_name: MemberAuthentication, dependent: :destroy
   has_many :payment_methods, through: :customer
+  has_many :actions
```

Comment:
Ohhh so the association already exists in the db for all the existing records, right? We had `action.member` just didn't have `member.actions`